### PR TITLE
Segemehl fix

### PR DESCRIPTION
**Pull request by jfallmann**
Fixed broken galaxy dependency, now conda only

### CODE REVIEW

**Comment by jfallmann** [comment]
@bgruening Sorry there is some old commit that belons to the aresite2 branch, whyever it made it's way into my local master, however, segemehl was not working out of galaxy due to a broken dependency (package_segemehl), with pure conda install it works now

**Comment by bgruening** [comment]
@jfallmann the commits are ok, I think but have you changed indentation of the wrapper? Can you please make sure you use 4 spaces intend? The diff is quite large and therefor hard to review.

File: tools/segemehl/tool_data_table_conf.xml.sample [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<!-- Use the file tool_data_table_conf.xml.oldlocstyle if you don't want to update your loc files as changed in revision 4550:535d276c92bc-->
```
Comment (bgruening) [inline tools/segemehl/tool_data_table_conf.xml.sample]: This file needs to stay ... it enables shared data resources between users.

**Comment by jfallmann** [comment]
@bgruening Sorry for the intent, was in the wrong emacs mode, should be fixed and the data_table_conf is back

**Comment by bgruening** [comment]
Thanks a lot!

**Comment by bgruening** [comment]
If you change parameters or add dependencies please increase the build number here as well - This way we get a new version of the tool in Galaxy.

**Pull request merged**